exchanges/bybit/bybit_test.go

```diff
@@ -3309,3 +3324,131 @@ func TestGetUSDCPredictedFundingRate(t *testing.T) {
+			if err != nil {
+				t.Fatalf("%v %v %v\n", avail[y], supportedAssets[x], err)
+			}
+		}
+	}
+}
+
+func TestGetTickersV5(t *testing.T) {
+	t.Parallel()
+
+	_, err := b.GetTickersV5(context.Background(), "bruh", "", "")
+	if err != nil && err.Error() != "Illegal category" {
+		t.Error(err)
+	}
+
+	_, err = b.GetTickersV5(context.Background(), "option", "", "")
+	if !errors.Is(err, errBaseNotSet) {
+		t.Fatalf("expected: %v, received: %v", errBaseNotSet, err)
+	}
+
+	_, err = b.GetTickersV5(context.Background(), "spot", "", "")
+	if err != nil {
+		t.Error(err)
+	}
+
+	_, err = b.GetTickersV5(context.Background(), "option", "", "BTC")
+	if err != nil {
+		t.Error(err)
+	}
+
+	_, err = b.GetTickersV5(context.Background(), "inverse", "", "")
+	if err != nil {
+		t.Error(err)
+	}
+
+	_, err = b.GetTickersV5(context.Background(), "linear", "", "")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestUpdateOrderExecutionLimits(t *testing.T) {
+	t.Parallel()
+
+	err := b.UpdateOrderExecutionLimits(context.Background(), asset.USDCMarginedFutures)
+	if !errors.Is(err, asset.ErrNotSupported) {
+		t.Fatalf("received: %v expected: %v", err, asset.ErrNotSupported)
+	}
+
+	err = b.UpdateOrderExecutionLimits(context.Background(), asset.Spot)
+	if err != nil {
+		t.Error("Okx UpdateOrderExecutionLimits() error", err)
+	}
+
+	avail, err := b.GetAvailablePairs(asset.Spot)
+	if err != nil {
+		t.Fatal("Okx GetAvailablePairs() error", err)
+	}
+
+	for x := range avail {
+		limits, err := b.GetOrderExecutionLimits(asset.Spot, avail[x])
+		if err != nil {
+			t.Fatal("Okx GetOrderExecutionLimits() error", err)
+		}
+		if limits == (order.MinMaxLevel{}) {
+			t.Fatal("Okx GetOrderExecutionLimits() error cannot be nil")
+		}
+	}
+}
+
+func TestGetAccountFee(t *testing.T) {
+	t.Parallel()
+
+	_, err := b.GetAccountFee(context.Background(), "", "", "")
+	if !errors.Is(err, errCategoryNotSet) {
+		t.Fatalf("received %v but expected %v", err, errCategoryNotSet)
+	}
+
+	_, err = b.GetAccountFee(context.Background(), "bruh", "", "")
+	if !errors.Is(err, errInvalidCategory) {
+		t.Fatalf("received %v but expected %v", err, errInvalidCategory)
+	}
+
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
+
+	_, err = b.GetAccountFee(context.Background(), "spot", "", "")
+	if !errors.Is(err, nil) {
+		t.Errorf("received %v but expected %v", err, nil)
+	}
+
+	_, err = b.GetAccountFee(context.Background(), "linear", "", "")
+	if !errors.Is(err, nil) {
+		t.Errorf("received %v but expected %v", err, nil)
+	}
+
+	_, err = b.GetAccountFee(context.Background(), "inverse", "", "")
+	if !errors.Is(err, nil) {
+		t.Errorf("received %v but expected %v", err, nil)
+	}
+
+	_, err = b.GetAccountFee(context.Background(), "option", "", "ETH")
+	if !errors.Is(err, nil) {
+		t.Errorf("received %v but expected %v", err, nil)
+	}
+}
+
+func TestForceFileStandard(t *testing.T) {
+	t.Parallel()
+	sharedtestvalues.ForceFileStandard(t, sharedtestvalues.EmptyStringPotentialPattern)
```

Comment:
I like this, but I think this test should return an error to log the specific reasons for failures within Bybit. Its a bit obtuse right now why its erroring. Yes it matches the pattern, but contributors may not understand why its failing and what they need to do to fix this and why we are preferring to use the new convert type. Please update it so it says not to use float64+ ",string" and instead use the convert type